src/lib/zAuction.ts

```diff
@@ -87,8 +103,41 @@ export async function getBidsForNft(
 	return bids;
 }
 
+export async function getBidsListForNfts(
+	baseApiUri: string,
+	contract: string,
+	tokenId: string[],
```

Comment:
`tokenId` implies a singular token id, but this is an array, thus `tokenIds` would be more appropriate here.